src/components/bodyImage.tsx

```diff
@@ -29,6 +30,11 @@ const imgStyles = (width: number, height: number): SerializedStyles => css`
     ${from.phablet} {
         height: calc(620px * ${height / width});
     }
+
+    ${darkModeCss`
+        color: ${neutral[60]};
+        background-color: ${neutral[20]};
+    `}
```

Comment:
Are these already covered by the changes you've made to the styles within the `Img` component?

Reply:
I think header image might be using `Img`

Reply:
Yeah it is, they both are.

As you've applied these styles within the `Img` component, are they needed here?